app/models/graph/schema.rb

```diff
@@ -23,8 +25,5 @@ module Graph
       model = Object.const_get(gid.model_name)
       Graph::FindLoader.for(model).load(gid.model_id.to_i)
     end
-
-    lazy_resolve(Promise, :sync)
-    instrument(:query, GraphQL::Batch::Setup)
```

Comment:
It does a `NestedError` 🤔. I've been trying to find a way to raise a useful error but since it's order dependant it's kinda hard :/